cmd/meroxa/root/apps/deploy.go

```diff
@@ -498,7 +497,7 @@ func (d *Deploy) getAppImage(ctx context.Context) (string, error) {
 	d.localDeploy.Lang = d.lang
 	d.localDeploy.AppName = d.appName
 	if d.localDeploy.Enabled {
-		fqImageName, err = d.localDeploy.GetDockerImageName(ctx, d.logger, d.path, d.appName, d.lang)
+		fqImageName, err = d.localDeploy.GetDockerImageName(ctx, d.logger, d.path, d.appName)
```

Comment:
Has usage of the `d.lang` parameter been removed from the `GetDockerImageName` function in the past and this is why it can be omitted from now on? Or is it better to revert this change?

Reply:
it is currently not used, and i'm not sure off the top of my head how it was ever used.  the linter seems to have released a new version that has a whole lot more complaints to deal with, mostly related to unused arguments.  this PR would not pass the linter check without making these changes.  my options were to replace the name of the argument with _ in order to indicate that it's intentionally not used or to remove the argument altogether.  this was one of the few cases in which the argument wasn't actually used at all, so it could just be removed.  when it comes to the Turbine interface, there's inconsistent use of arguments, so it makes more sense to just unname those arguments.

Reply:
I see and thank you for the outline on the linter upgrade leading to this change now and not previously, including this fix makes sense to me and thank you for the upgrade!